src/main/java/edu/harvard/iq/dataverse/util/bagit/BagGenerator.java

```diff
@@ -816,7 +824,7 @@ private String generateInfoFile() {
         } else {
             info.append(
                     // FixMe - handle description having subfields better
-                    WordUtils.wrap(getSingleValue(aggregation.getAsJsonObject(descriptionTerm.getLabel()),
+                    WordUtils.wrap(getSingleValue(aggregation.get(descriptionTerm.getLabel()),

```

Comment:
Should this be getAsString?

Reply:
Description is a multi-valued field so this could be a JsonObject or JsonArray. The added logic here handles both cases. Nominally this is also separable from the API itself - just caught in testing along the way. Could be moved to a separate PR.